src/objectivec/test/model_tests.mm

```diff
@@ -0,0 +1,125 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <XCTest/XCTest.h>
+
+#import "ort_genai_objc.h"
+#import "assertion_utils.h"
+#import <vector>
+#import <array>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ORTGenAIModelTest : XCTestCase
+
+@end
+
+@implementation ORTGenAIModelTest
+
+
+- (void)GreedySearchGptFp32 {
```

Comment:
do test methods need to start with "test"?

https://developer.apple.com/documentation/xctest/defining_test_cases_and_test_methods?language=objc

Reply:
yeah. thanks for catching this. i forgot when i modified it, and since then the test are actually not run (just compiles). And because I haven't set up the testing pipeline yet, i didn't notice it until now.